site/reference/pkg/update/README.md

```diff
@@ -67,33 +43,45 @@ VERSION:
 
 ```
 --strategy:
-  Controls how changes to the local package are handled.  Defaults to fast-forward.
+  Defines which strategy should be used to update the package. This will change
+  the update strategy for the current kpt package for future updates. If a
+  strategy is not provided, the strategy specified in the package Kptfile will
+  be used.
 
     * resource-merge: perform a structural comparison of the original /
-      updated Resources, and merge the changes into the local package.
+      updated resources, and merge the changes into the local package.
     * fast-forward: fail without updating if the local package was modified
       since it was fetched.
-    * alpha-git-patch: use 'git format-patch' and 'git am' to apply a
-      patch of the changes between the source version and destination
-      version.
-    * force-delete-replace: WIPE ALL LOCAL CHANGES TO THE PACKAGE.
-      DELETE the local package at local_pkg_dir/ and replace it
-      with the remote version.
-
--r, --repo:
-  Git repo url for updating contents.  Defaults to the repo the package
-  was fetched from.
-
---dry-run
-  Print the 'alpha-git-patch' strategy patch rather than merging it.
+    * force-delete-replace: Wipe all the local changes to the package and replace
+      it with the remote version.
 ```
 
 #### Env Vars
 
 ```
 KPT_CACHE_DIR:
-  Controls where to cache remote packages when fetching them to update
-  local packages.
+  Controls where to cache remote packages when fetching them.
   Defaults to ~/.kpt/repos/
 ```
 <!--mdtogo-->
+
+### Examples
+<!--mdtogo:Examples-->
+```shell
+# update my-package-dir/
+git add . && git commit -m 'some message'
+kpt pkg update my-package-dir/
```

Comment:
for all commands, provide an example where the package name is omitted. This is the shortest/common usage.

Reply:
Fixed.